src/components/Avatar/Avatar.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from 'react';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import InitialsPropType from '../../prop-types/initials';
+import ListPropType from '../../prop-types/list';
+import './Avatar.scss';
+
+const propTypes = {
+    name: PropTypes.string.isRequired,
+    initials: InitialsPropType,
+    src: PropTypes.string,
+    size: ListPropType(['xs', 'sm', 'md', 'lg']),
+    theme: ListPropType(['light', 'dark']),
+    status: PropTypes.string,
+    hasNotification: PropTypes.bool,
+    hasHalo: PropTypes.bool,
+    onClick: PropTypes.func,
+    tabIndex: PropTypes.number,
+    className: PropTypes.string,
+    style: PropTypes.objectOf(PropTypes.oneOfType([
+        PropTypes.string,
+        PropTypes.number,
```

Comment:
I like this (if it works fine for all cases), we probably should do the same style Proptypes for all style props, including Button

Reply:
@jfrej found this, I believe it's been added to the Button too 👍

Reply:
It works but it doesn't really validate anything - it just checks if the `style` prop is an object and allows both string and number as object key values. So basically allows anything inside the object.
If we wanted proper `style` validation, we should use https://www.npmjs.com/package/react-style-proptype